Fritz.StreamTools/Views/Followers/PreviewGoal.cshtml

```diff
@@ -101,20 +154,30 @@
 
 			function onload() {
 
-				var elements = document.getElementsByTagName("input");
-				for (var el of elements) {
+				for (var i = 0; i < localStorage.length; i++) {
+					var key = localStorage.key(i);
+					var item = localStorage.getItem(key);
+					console.log(key,item);
 
-					console.log(el.id);
-
-					if (localStorage.getItem(el.id)) {
-						el.value = localStorage.getItem(el.id);
+					// make sure we create all the required color pickers
+					if (key.substr(0, 2) == "bg") {
+						if (!document.getElementById(key)) {
+							addColor(i+1);
+						}
 					}
+
+					var el = document.getElementById(key);
+					if (el) {
+						el.value = item;
+					}					
 				}
 
 			}
 
 			function saveValues() {
 
+				localStorage.clear();
```

Comment:
This is here to prevent saved values coming back even if we have removed the item from the page (this applies to colors/blends).